lib/Db/SessionMapper.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
```suggestion
<?php

declare(strict_types=1);
```

Would be good to add that to all new classes.